benchmarks/benchmark_functions/pl-braket-pipeline2.py

```diff
@@ -0,0 +1,127 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Fixed benchmarks for pennylane-braket pipelines.
+"""
+from numpy.random import random
+import pennylane as qml
+import networkx as nx
+from pennylane import numpy as pnp
+from pennylane import qaoa
+
+def benchmark_casual(dev_name, s3=None):
+    """ A simple optimization workflow
+
+    Args:
+        dev_name (str): Either "local", "sv1", "tn1", or "ionq"
+        s3 (tuple):  A tuple of (bucket, prefix) to specify the s3 storage location
+
+    """
+    n_steps = 2
+    n_wires = 4
```

Comment:
Let us set the wires per device:

Local, sv1, tn1 - 20 (see discussion on slack - but I think we should open the PR to others for comments since this is crucial)

IonQ - 11